modules/ROOT/pages/create-partner.adoc

```diff
@@ -47,7 +47,7 @@ Add the following settings for the new partner, either now or later on:
 | Partner-specific endpoints to use for sending and receiving messages to and from your partner, including acknowledgement endpoints. If you plan to use host endpoints only, you don't need to create partner-specific endpoints.
 | Click *New* in the *Endpoints* section.
 | xref:create-endpoint.adoc[Creating Endpoints]
-| Acknowledgements, validations, and control numbers for each message format that the partner uses. For X12, you can also configure character sets and separators (X12 send format only).
+| Acknowledgements, validations, and control numbers for each message format that the partner uses. For EDIFACT and X12, you can also configure character sets and separators (send formats only).
```

Comment:
Acknowledgments

Should we add that to vale?

Reply:
This is a situation in which the GUI uses the British spelling so I want to leave what I have.